docs/UserGuide.md

```diff
@@ -126,7 +126,6 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find John` returns `john` and `John Doe`
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
```

Comment:
We should probably remove that file if we are not going to be using it.

Reply:
File removed.